### DISCUSSION TITLE
Discord Discussion

### DISCUSSION THREAD

**Discussion by r-colvin** [outline/outline#4072] [General]
Hi!

I have only just discovered Outline, and it looks super impressive; it fits about 90% of my requirements - being open source I am hoping I can solve the remaining 10% somehow. 

For this reason it would be great to have a discord where users can chat - developers can exchange ideas, community can provide support etc. It would not be a replacement for Discuss - but chatting with other developers, rubber duckying ideas etc doesn't seem like a good fit for Discuss

would anyone else be interested in hanging out on a Discord, or am I in the minority?!

I have created a server just in case... but let me know if it won't be just me screaming into the void :)

**Comment by Limezy** [comment]
Meanwhile, let's use the GitHub discussions 😀 
By curiosity, could you give us a hint about these missing 10% ? Aren't they in the current roadmap ? Could they be interesting ideas to be included in it ?

**Reply by r-colvin** [reply]
@Limezy  happy to use the Discuss if that's the best option :) offering Discord as an alternative for more general discussion

I am a big fan of one-commit = one change/topic and not letting discussions wander into other areas - given the topic of this is "Discord Discussion" shall i start a new topic instead to discuss the missing 10%? :) 

have started a new topic to discuss my needs here: https://github.com/outline/outline/discussions/4074
